src/views/AddItem.jsx

```diff
@@ -25,18 +26,25 @@ export function AddItem() {
 		setFormData((formData) => ({ ...formData, [name]: value }));
 	};
 
+	const itemNames = data.map((item) =>
+		item.name.toLowerCase().replace(/[^a-z0-9]/gi, ''),
```

Comment:
I'm not very familiar with regex, so the .replace(/[^a-z0-9]/gi, '') was really cool to see! Really neat way to handle case insensitivity.